From 734ae968a10bdf9029eabaca59af1e02220aa937 Mon Sep 17 00:00:00 2001
From: Kay Shinonome <kayshinonome@protonmail.com>
Date: Tue, 12 Apr 2022 03:43:05 -0500
Subject: [PATCH 3/5] force-dynamic-http-parser-linkage

---
 make-linux.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/make-linux.mk b/make-linux.mk
index ab79432a..78d82938 100644
--- a/make-linux.mk
+++ b/make-linux.mk
@@ -43,9 +43,9 @@ endif
 LDLIBS+=-lnatpmp
 override DEFS+=-DZT_USE_SYSTEM_NATPMP
 
-# Use bundled http-parser since distribution versions are NOT API-stable or compatible!
-# Trying to use dynamically linked libhttp-parser causes tons of compatibility problems.
-ONE_OBJS+=ext/http-parser/http_parser.o
+# We have to use http-parser here
+LDLIBS+=-lhttp_parser
+override DEFS+=-DZT_USE_SYSTEM_HTTP_PARSER
 
 ifeq ($(ZT_RULES_ENGINE_DEBUGGING),1)
 	override DEFS+=-DZT_RULES_ENGINE_DEBUGGING
-- 
2.35.1

