From 7f61a54d49b9c9bc40a05350ac256305e90d6592 Mon Sep 17 00:00:00 2001
From: Kay Shinonome <kayshinonome@protonmail.com>
Date: Tue, 12 Apr 2022 04:21:24 -0500
Subject: [PATCH 5/5] respect-ld-flags

---
 make-linux.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/make-linux.mk b/make-linux.mk
index 9d63df7b..7172dfa5 100644
--- a/make-linux.mk
+++ b/make-linux.mk
@@ -72,7 +72,7 @@ else
 	override CFLAGS+=-Wall -Wno-deprecated -pthread $(INCLUDES) -DNDEBUG $(DEFS)
 	CXXFLAGS?=-O3 -fstack-protector -fPIE
 	override CXXFLAGS+=-Wall -Wno-deprecated -std=c++11 -pthread $(INCLUDES) -DNDEBUG $(DEFS)
-	LDFLAGS=-pie -Wl,-z,relro,-z,now
+	LDFLAGS?=-pie -Wl,-z,relro,-z,now
 	RUSTFLAGS=--release
 endif
 
-- 
2.35.1

